sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/command.py

```diff
@@ -277,12 +281,12 @@ def distribution(
         self._distribution = value
 
     @property
-    def resources(self) -> Any:
+    def resources(self) -> JobResourceConfiguration:
         """The compute resource configuration for the command component or job.
 
         :rtype: ~azure.ai.ml.entities.JobResourceConfiguration
         """
-        return self._resources
+        return self._resources  # type: ignore[return-value]
```

Comment:
Looks like this could be fixed with `cast(JobResourceConfiguration, self._resources)` (assuming that we know that this _is_ an instance of `JobResourceConfiguration`.....